src/content-scripts/components/AddPRToHighlights/AddPRToHighlightsButton.ts

```diff
@@ -10,7 +10,9 @@ export const AddPRToHighlightsButton = () => {
             openSaucedLogoIcon,
         )}>
     <details-menu id="details-menu-os" class="dropdown-menu hidden dropdown-menu-sw color-fg-default w-48 mt-2">
-    <a href="https://${OPEN_SAUCED_INSIGHTS_DOMAIN}/feed?prurl=${encodeURIComponent(window.location.href)}" class="dropdown-item" target="_blank">
+    <a href="https://${OPEN_SAUCED_INSIGHTS_DOMAIN}/feed?prurl=${encodeURIComponent(
```

Comment:
```suggestion
    <a href="https://${OPEN_SAUCED_INSIGHTS_DOMAIN}/feed?new=true&prurl=${encodeURIComponent(
```
RThis is my suggestion